.github/workflows/node.js.yml

```diff
@@ -179,25 +114,13 @@ jobs:
     # == check precommit works ==
     - run: yarn run precommit
 
-    - name: Report Failure to Slack
```

Comment:
Hmm, some might protest removing the Slack notifications. I don't think they're a perfect solution, but I agree we need to have some kind of monitoring and alerts on broken builds.

Reply:
We have already removed them in most other repos. Its replaced by other monitoring. This is a leftover